### PR TITLE
US616059 Multi-valued metadata support

### PR DESCRIPTION
**Pull request by kploch**
https://internal.almoctane.com/ui/entity-navigation?p=131002/6001&entityType=work_item&id=616059

### CODE REVIEW

File: pom.xml [hunk 1]
```diff
@@ -101,12 +101,12 @@
             <dependency>
                 <groupId>io.github.fileanalysissuite.adaptersdk.convenience</groupId>
                 <artifactId>adaptersdk-convenience</artifactId>
-                <version>1.0.0-SNAPSHOT</version>
+                <version>1.0.0-US616059MultiValue-SNAPSHOT</version>
```
Comment (kploch) [inline pom.xml]:
```suggestion
                <version>1.0.0-SNAPSHOT</version>
```

File: pom.xml [hunk 2]
```diff
@@ -101,12 +101,12 @@
...
             </dependency>
             <dependency>
                 <groupId>io.github.fileanalysissuite.adaptersdk.interfaces</groupId>
                 <artifactId>adaptersdk-interfaces</artifactId>
-                <version>1.0.0-SNAPSHOT</version>
+                <version>1.0.0-US616059MultiValue-SNAPSHOT</version>
```
Comment (kploch) [inline pom.xml]:
```suggestion
                <version>1.0.0-SNAPSHOT</version>
```

File: pom.xml [hunk 3]
```diff
@@ -354,7 +354,7 @@
                     <dependency>
                         <groupId>io.github.fileanalysissuite.adapters.rest.contract</groupId>
                         <artifactId>adapter-rest-contract</artifactId>
-                        <version>1.0.0-SNAPSHOT</version>
+                        <version>1.0.0-US616059MultiValue-SNAPSHOT</version>
```
Comment (kploch) [inline pom.xml]:
```suggestion
                        <version>1.0.0-SNAPSHOT</version>
```

File: src/main/java/io/github/fileanalysissuite/adaptersdk/impls/jaxrs/FileMetadataImpl.java [hunk 4]
```diff
@@ -82,9 +80,9 @@ public Integer getVersion()
 
     @Nonnull
     @Override
-    public Map<String, Serializable> getAdditionalMetadata()
+    public Map<String, Iterable<String>> getAdditionalMetadata()
     {
-        final Map<String, String> additionalMetadata = metadata.getAdditionalMetadata();
+        final Map<String, Iterable<String>> additionalMetadata = metadata.getAdditionalMetadata().entrySet().stream().collect(Collectors.toMap(Map.Entry::getKey, Map.Entry::getValue));
```
Comment (dermot-hardy) [inline src/main/java/io/github/fileanalysissuite/adaptersdk/impls/jaxrs/FileMetadataImpl.java]: Simplify this in light of interface changes.

**Review by dermot-hardy — CHANGES_REQUESTED** [review]

**Review by dermot-hardy — APPROVED** [review]

**Pull request merged**